joplin/base/tests/test_incremental_builds.py

```diff
@@ -56,6 +57,7 @@ def test_information_page_in_guide_page(self):
         global_page_id = Node.to_global_id(permit_guide.get_verbose_name().lower(), permit_guide.id)
         self.assertTrue(global_page_id in global_ids)
 
+    @pytest.mark.xfail
```

Comment:
this lets pytest know that the test will fail, but we expect it to. 
tests can also be skipped based on a variety of criteria:
https://docs.pytest.org/en/latest/skipping.html